setup.py

```diff
@@ -0,0 +1,21 @@
+"""
+    Setup file for orca.
+    Use setup.cfg to configure your project.
+
+    This file was generated with PyScaffold 4.3.
+    PyScaffold helps you to put up the scaffold of your new Python project.
+    Learn more under: https://pyscaffold.org/
+"""
+from setuptools import setup
+
+if __name__ == "__main__":
+    try:
+        setup(use_scm_version={"version_scheme": "no-guess-dev"})
+    except:  # noqa
+        print(
+            "\n\nAn error occurred while building the project, "
+            "please ensure you have the most updated version of setuptools, "
```

Comment:
Might be know what version of setuptools it should at least be

Reply:
This came with the PyScaffold template, so I don't know what is the minimum version for this package.